src/planwise/client/scenarios/views.cljs

```diff
@@ -44,27 +44,29 @@
   (not (nil? (:action provider))))
 
 (defn- show-provider
-  [{{:keys [action name capacity satisfied unsatisfied investment]} :elem :as ix-provider}]
+  [{{:keys [action name capacity free-capacity required-capacity satisfied unsatisfied investment]} :elem :as ix-provider}]
   (str "<b>" (utils/escape-html (if (provider-from-changeset? (:elem ix-provider))
                                   (str "New provider " (:index ix-provider))
                                   name)) "</b>"
        "<br> Capacity: " capacity
        "<br> Satisfied demand: " satisfied
        "<br> Unsatisfied demand: " unsatisfied
-       "<br> Current capacity: " (- capacity satisfied)
+       "<br> Free capacity: " (.toFixed (or free-capacity 0) 2)
```

Comment:
We have `planwise.client.utils/format-number` which will do this and add thousands separators as well.

Reply:
Please, add the formatting to satisfied and unsatisfied demand as well.